judge/create_cgroups.in

```diff
@@ -9,6 +9,14 @@
 JUDGEHOSTUSER=@DOMJUDGE_USER@
 CGROUPBASE="/sys/fs/cgroup"
 
+# We do not need to do any of this with cgroup v2.
```

Comment:
This comment should probably be changed to something like:
```suggestion
# Check whether we have cgroup v2 available.
```

Reply:
Done